src/renderer/screens/exchange/Buy/SelectProvider.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import React from "react";
+import styled, { css } from "styled-components";
+import { useTranslation } from "react-i18next";
+
+import Text from "~/renderer/components/Text";
+import Box from "~/renderer/components/Box";
+import { rgba } from "~/renderer/styles/helpers";
+import Bullet from "~/renderer/screens/exchange/Buy/Bullet";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+const Container: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  alignItems: "center",
+  cursor: "pointer",
+  boxShadow: p.isActive ? `0 0 0 4px ${rgba(p.theme.colors.palette.primary.main, 0.25)}` : "",
+}))`
+  border: 1px solid ${p => p.theme.colors.palette.divider};
+  border-radius: 4px;
+  width: calc(50% - 12px);
```

Comment:
preferably use flex layout with `row-gap` to handle 50:50 layout with spacing in between

Reply:
I assume you meant `column-gap`. Yeah, reworked it. Thank you!